### PR TITLE
[12.0][FIX] base_transaction_id template extension not working

### PR DESCRIPTION
**Pull request by mpanarin**
and breaking the whole view

### CODE REVIEW

File: base_transaction_id/static/src/xml/account_reconciliation.xml [hunk 1]
```diff
@@ -2,10 +2,24 @@
...
+          <t t-if="line.partner_id &amp;&amp; line.partner_id !== state.st_line.partner_id">
+            <t t-if="line.partner_name.length">
+              <t t-esc="line.partner_name"/>:
+            </t>
+          </t>
+          <t t-esc="line.label || line.name"/>
+          <t t-if="line.ref &amp;&amp; line.ref.length"> : </t>
+          <t t-esc="line.ref"/>
+        <!-- Added line here -->
+        <t t-if="line.transaction_ref"> (<t t-esc="line.transaction_ref"/>)</t>
```
Comment (simahawk) [inline base_transaction_id/static/src/xml/account_reconciliation.xml]: bad indent

File: base_transaction_id/static/src/xml/account_reconciliation.xml [hunk 2]
```diff
@@ -2,10 +2,24 @@
 
 <templates xml:space="preserve">
 
-  <t t-name="reconciliation.line.mv_line" t-inherit="account.reconciliation.line.mv_line">
-    <xpath expr="//td[@class='cell_label']" position="inside">
-      <t t-if="line.transaction_ref"> (<t t-esc="line.transaction_ref"/>)</t>
-    </xpath>
+  <t t-extend="reconciliation.line.mv_line">
+    <t t-jquery=".cell_label" t-operation="replace">
+      <!-- I am replacing this, as there is no component inside, that I can find with jquery -->
```
Comment (simahawk) [inline base_transaction_id/static/src/xml/account_reconciliation.xml]: what do you mean? `t[t-esc="line.ref"]` should do the trick

https://api.jquery.com/attribute-equals-selector/

**Review by simahawk — CHANGES_REQUESTED** [review]

File: base_transaction_id/static/src/xml/account_reconciliation.xml (hunk 2 above)
Comment (mpanarin) [reply base_transaction_id/static/src/xml/account_reconciliation.xml]: updated

**Review by simahawk — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Review by Tonow-c2c — APPROVED** [review]
:+1:

**Pull request merged**